app/models/periodic_expense_runner.rb

```diff
@@ -27,12 +27,6 @@ def update_periodic_expense(periodic_expense)
   end
 
   def create_expense(periodic_expense)
-    Expense.create!(
-      name: "#{periodic_expense.name} (#{periodic_expense.period.downcase})",
-      user: periodic_expense.user,
-      periodic_expense: periodic_expense,
-      paid_on: today.to_datetime,
-      amount: periodic_expense.amount
-    )
+    periodic_expense.create_expense.save
```

Comment:
Not really. `create` will save ir to the database. `new` is the method that doesn't touch the database. The difference between `create` and `create!` is that one returns `false` and the other an exception in the case of an error.